test/core/public/run.js

```diff
@@ -211,6 +226,24 @@ describe('axe.run', function () {
 				done();
 			});
 		});
+
+		it('is called after cleanup',
+		(!window.Promise) ? undefined :  function (done) {
```

Comment:
This condition is rather difficult to follow. It may be nicer to write this as:

```js
const testWhenPromiseIsSupported = window.Promise ? it : it.skip
testWhenPromiseIsSupported('is called after cleanup', () => {
  // ...
})
```

Reply:
Alternately, we could just add a Promise polyfill :wink: